tensorflow/models/rnn/ptb/ptb_word_lm.py

```diff
@@ -130,8 +130,7 @@ def __init__(self, is_training, config, input_):
     #
     # The alternative version of the code below is:
     #
-    # inputs = [tf.squeeze(input_step, [1])
-    #           for input_step in tf.split(1, num_steps, inputs)]
+    # inputs = tf.unstack(inputs, num_steps, 1)
```

Comment:
One more suggestion: might be nice to use named arguments here for clarity:

num=num_steps, axis=1